packages/tasks/src/snippets/python.ts

```diff
@@ -17,6 +17,30 @@ for message in client.chat_completion(
 ):
     print(message.choices[0].delta.content, end="")`;
 
+export const snippetConversationalWithImage = (model: ModelDataMinimal, accessToken: string): string =>
+	`from huggingface_hub import InferenceClient
+
+client = InferenceClient(
+	"${model.id}",
+	token="${accessToken || "{API_TOKEN}"}",
+)
+image_url = "https://cdn.britannica.com/61/93061-050-99147DCE/Statue-of-Liberty-Island-New-York-Bay.jpg"
```

Comment:
maybe good to show an example with `pillo.Image.open` and use that image's base64 so that users can get an example where they can load local images

Reply:
hmm i'd say this would complicate a bit too much (no strong opinion though)

Note that this is for remote inference not local usage

Reply:
do you know if it's possible to have several snippets by returning a list, same as for code snippets ?

Reply:
> do you know if it's possible to have several snippets by returning a list, same as for code snippets ?

for inference snippet, not possible right now. So suggest that:
1. we merge this PR as it is with only image url example
2. mayeb unify on moon-side to have inference snippet to be able to have a list like code snippets. If so, we can re-iterate and add an example with a local image